kodiak/evaluation.py

```diff
@@ -153,10 +153,17 @@ class PRAPI(Protocol):
         ...
 
 
-async def cfg_err(api: PRAPI, pull_request: PullRequest, msg: str) -> None:
+async def cfg_err(
+    api: PRAPI,
+    pull_request: PullRequest,
+    msg: str,
+    markdown_content: Optional[str] = None,
```

Comment:
docs for what `markdown_content` is compared to `msg` and where it appears in the GitHub ui?

Reply:
markdown content is what's shown on the page when you click "Details".